

Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:23:49 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:23:51 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:23:53 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:23:56 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:23:59 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:02 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:05 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:07 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:09 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:11 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:14 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:16 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:18 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:21 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:24 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:27 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:30 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:33 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:36 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:39 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:41 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:44 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:47 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:50 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/sidekiq/dashboard/stats" for 127.0.0.1 at 2014-11-01 01:24:53 +0800

Errno::ENOENT (No such file or directory - /Applications/web_apps/sidekiq_mailer/config/config/database.yml):
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `read'
  railties (3.2.16) lib/rails/application/configuration.rb:115:in `database_configuration'
  activerecord (3.2.16) lib/active_record/railtie.rb:84:in `block (2 levels) in <class:Railtie>'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `instance_eval'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:36:in `execute_hook'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:43:in `block in run_load_hooks'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `each'
  activesupport (3.2.16) lib/active_support/lazy_load_hooks.rb:42:in `run_load_hooks'
  activerecord (3.2.16) lib/active_record/base.rb:720:in `<top (required)>'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:483:in `rescue in call'
  activerecord (3.2.16) lib/active_record/connection_adapters/abstract/connection_pool.rb:477:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `_run__4098875154531552486__call__2421449441505809746__callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.16) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.16) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.16) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.16) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.16) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.16) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.16) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.16) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.16) lib/rails/engine.rb:484:in `call'
  railties (3.2.16) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.16) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.5) lib/rack/handler/webrick.rb:59:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  /Users/choyno/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


